### ISSUE TITLE
Schedule Race is not working on v2.2.0-dev.7

### ISSUE THREAD

**Issue by Klaus-Michael** [RotorHazard/RotorHazard#342]
The "Schedule Race" function in the Race page is no longer working for me.
When pressing the button I get the message that the next race will start in XX seconds, but nothing else is happening. No voice Callouts and the Race is not started.

**Comment by HazardCreative** [comment]
Works for me. Can you provide more information so I can reproduce?

**Comment by Klaus-Michael** [comment]
I cleaned up my Rotorhazard directory and started from scratch with the latest bits from the main branch. Updated everything with apt-get and got the latest requirements to be sure.

I Simply press the Schedule Race button. 
With this fresh installation after 15s the race is startet correctly. But no Voice Callout is happening.

Gave it a try with two different PC's and my android Phone. Trying Chrome & Edge (on the PC)
Cleaned up all cache data in the Browsers prior to trying. 

The previous error that it was not working at all seems to be a issue if no Pilots are added to the Heat the server crash's hard with a error:

Exception in Heartbeat thread loop:  Working outside of request context.

This typically means that you attempted to use functionality that needed
an active HTTP request.  Consult the documentation on testing for
information about how to avoid this problem.
Traceback (most recent call last):
  File "server.py", line 4086, in heartbeat_thread_function
    on_stage_race()
  File "server.py", line 2026, in on_stage_race
    emit_priority_message(__('No valid pilots in race'), True, nobroadcast=True)
  File "server.py", line 2797, in emit_priority_message
    emit('priority_message', emit_payload)
  File "/usr/local/lib/python2.7/dist-packages/flask_socketio/__init__.py", line 752, in emit
    namespace = flask.request.namespace
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/local.py", line 348, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/local.py", line 307, in _get_current_object
    return self.__local()
  File "/usr/local/lib/python2.7/dist-packages/flask/globals.py", line 38, in _lookup_req_object
    raise RuntimeError(_request_ctx_err_msg)
RuntimeError: Working outside of request context.
